src/ebay-inline-notice/__tests__/__snapshots__/index.spec.tsx.snap

```diff
@@ -131,7 +131,7 @@ exports[`Storyshots ebay-inline-notice Notice toggle 1`] = `
     Hide Notice
   </button>
   <div
-    class="inline-notice inline-notice--confirmation"
+    class="inline-notice inline-notice--\${status}"
```

Comment:
This should be `confirmation` not `${status}` this is because of using single quote instead of backtick (comment bellow)